docs/ref/csrf.txt

```diff
@@ -132,6 +132,12 @@ The above code could be simplified by using the `JavaScript Cookie library
     the DOM. Regardless, you're guaranteed to have the cookie if the token is
     present in the DOM, so you should use the cookie!
 
+.. note::
+
+    If you activate :setting:`CSRF_USE_SESSIONS`, this method does not work and
+    need to make sure you always include the CSRF token somewhere in your HTML
+    read the token from the DOM with JavaScript.
+
```

Comment:
On second thought, should we provide the token in a cookie even when sessions are used, to support AJAX?

Or should we provide an AJAX-friendly version of `{% csrf_token %}` which creates a data element rather than a hidden form field?